tests/test_e2e.py

```diff
@@ -1,24 +1,23 @@
-import asyncio
 from collections.abc import Generator
 from concurrent.futures import ThreadPoolExecutor
 from pathlib import Path
+import subprocess
 import sys
+from time import sleep
 
 from click.testing import CliRunner
 from freezegun import freeze_time
 import httpx
 import pytest
-from pytest_mock import MockerFixture
 
 from imagedephi import main
-from imagedephi.utils.network import wait_for_port
 
 
 @pytest.fixture
 def thread_executor() -> Generator[ThreadPoolExecutor, None, None]:
```

Comment:
If you're not using a fixture, maybe it should be deleted, since otherwise it's mysterious cruft that everyone else will be afraid to delete too.

Reply:
removed in c95f49ee9c4caa7ae50536b73b038fcb266dcc81